e2e/page-objects/entries-page-object.ts

```diff
@@ -19,7 +19,9 @@ export class EntriesPageObject {
   entriesThirdMetaDataField: ElementFinder = this.entriesFields.get(2);
   entriesCameraButton: ElementFinder = this.entriesItem.element(by.css('.cameraButton'));
   entriesGalleryButton: ElementFinder = this.entriesItem.element(by.css('.galleryButton'));
+  entriesFileUpload: ElementFinder = this.entriesItem.element(by.css('.fileUpload'));
 
+  path = require('path');
```

Comment:
Maybe this could be done with a `import * as Path from 'path';` instead, perhaps directly in `helpers.ts`?